sdk/provisioning/Azure.Provisioning.EventGrid/src/EventGridDomain.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Provisioning.EventGrid;
+
+/// <summary>
+/// EventGridDomain.
+/// </summary>
+public partial class EventGridDomain
+{
+    /// <summary>
+    /// The Sku name of the resource. The possible values are: Basic or Premium.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public BicepValue<EventGridSku> SkuName
+    {
+        get => throw new NotSupportedException("TODO: Needs to be implemented using extensibility API.");
+        set => throw new NotSupportedException("TODO: Needs to be implemented using extensibility API.");
+    }
```

Comment:
Emmm would we need follow ups for this before our next release?
It makes me kind of uncomfortable to update an existing API to "not working"

Reply:
These will get patched to use the new dynamic API in the next couple of days.  Leaving former preview APIs slightly broken for a couple of days is worth it to avoid more aggressive refactoring of property definition to make it hackable from partial classes.